apis/core/lib/domain/cube-projects/create.ts

```diff
@@ -26,30 +26,35 @@ export async function createProject({
   if (!label) {
     throw new Error('Missing project name')
   }
-  const publishGraph = resource.out(cc.publishGraph).term
-  if (!publishGraph || publishGraph.termType !== 'NamedNode') {
-    throw new DomainError('Missing publish graph or not a named node')
+  const maintainer = resource.out(schema.maintainer).term
+  if (!maintainer || maintainer.termType !== 'NamedNode') {
+    throw new DomainError('Missing organization or not a named node')
+  }
+  const cubeIdentifier = resource.out(dcterms.identifier).value
+  if (!cubeIdentifier) {
+    throw new Error('Missing cube identifier name')
   }
 
   const project = Project.create(await store.createMember(projectsCollection.term, id.cubeProject(label)), {
     creator: user,
     label,
-    publishGraph,
+    maintainer,
+    cubeIdentifier,
   })
 
   project.initializeJobCollection(store)
-  const dataset = Dataset.create(store.create(project.dataset.id))
+  const dataset = Dataset.create(store.create(project.dataset.id), {
+    [dcterms.identifier.value]: project.cubeIdentifier,
```

Comment:
The `identifier` field in the dataset form needs a description. I find it extremely confusing that we have 2 fields named "identifier". It's even more confusing now that the dataset one defaults to being the same value as the cube one, but when I change one it doesn't change the other.

Reply:
Any suggestions? I don't know what I would write there

Reply:
Not really because I myself don't understand the difference

Reply:
The value you provide when creating the project is used for the URL and initial metadata value

When editing, those values are separated because the URL has to be stricter than the metadata identifier (pretty much any string allowed I guess)

Reply:
Ok but what is the dataset one used for? That's what we need to explain in the UI

Reply:
Everything about the dataset is just metadata. Stuff needed to be published with the cube. Not unlike any other of those dataset properties

Reply:
Metadata still has an effect somewhere, or a usefulness for someone. We need to explain to users "why" they have to provide this value, and what will happen with it (if applicable). Is it the "cube identifier on opendata.swiss"? Or something like that?

Reply:
https://www.w3.org/TR/vocab-dcat-2/#Property:resource_identifier seems to be optional first of all.

Is it possible to change the "project" identifier? If so I would actually remove the dataset.identifier and re-use the "project" identifier. Because to have a dataset with a URI and then added another Identifier will be confusing.

Reply:
Will the "metadata" identifier prepopulated in the current version?